### PR TITLE
[new release] reparse, reparse-lwt and reparse-lwt-unix (3.0.1)

### PR DESCRIPTION
**Pull request by bikallem**
Recursive descent parsing library for ocaml

- Project page: <a href="https://github.com/lemaetech/reparse">https://github.com/lemaetech/reparse</a>

##### CHANGES:

- Fix `take_while`

### CODE REVIEW

File: packages/reparse-lwt/reparse-lwt.3.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+authors: ["Bikal Lem <gbikal@gmail.com>"]
+license: "MPL-2.0"
+homepage: "https://github.com/lemaetech/reparse"
+bug-reports: "https://github.com/lemaetech/reparse/issues"
+depends: [
+  "dune" {>= "2.8"}
+  "ocaml" {>= "4.10.0"}
+  "lwt" {>= "5.4.1"}
+  "cstruct" {>= "6.0.0"}
+  "reparse" {>= "3.0.0"}
```
Comment (mseri) [inline packages/reparse-lwt/reparse-lwt.3.0.1/opam]:
```suggestion
  "reparse" {= version}
```

File: packages/reparse-lwt-unix/reparse-lwt-unix.3.0.1/opam [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+authors: ["Bikal Lem <gbikal@gmail.com>"]
+license: "MPL-2.0"
+homepage: "https://github.com/lemaetech/reparse"
+bug-reports: "https://github.com/lemaetech/reparse/issues"
+depends: [
+  "dune" {>= "2.8"}
+  "ocaml" {>= "4.10.0"}
+  "lwt" {>= "5.4.1"}
+  "cstruct" {>= "6.0.0"}
+  "reparse" {>= "3.0.0"}
```
Comment (mseri) [inline packages/reparse-lwt-unix/reparse-lwt-unix.3.0.1/opam]:
```suggestion
  "reparse" {= version}
```

**Review by mseri — COMMENTED** [review]
When releasing a bundle of packages we tend to be strict with dependencies version, this has shown to improve the maintainability and release process quite a lot. Do you mind if we change them as suggested?

**Comment by mseri** [comment]
I would still recommend making the releases stricter if you don't mind, even if the sub-packages are unchanged. It used to become a nightmare fairly rapidly in the past (I will also ping @kit-ty-kate to check her opinion on this)

**Comment by bikallem** [comment]
> I would still recommend making the releases stricter if you don't mind, even if the sub-packages are unchanged.
Do you mean using `"reparse" {= version}` in `reparse-lwt.opam` and `reparse-lwt-unix.opam` files or always releasing the other opam packages in the repo - even though the packages hasn't changed?

**Comment by mseri** [comment]
Yes, and re-relasing them in block each time (so the 3.0.1 tagged source bundle refers to version 3.0.1 of _all_ the packages).

**Comment by mseri** [comment]
For example: https://github.com/ocaml/opam-repository/pull/18940

**Comment by bikallem** [comment]
@mseri Made the changes you requested. Hopefully it is a pass. :)

**Comment by mseri** [comment]
Thanks a lot! This is very appreciated

**Comment by mseri** [comment]
I'll let the CI test it again, but it was green before so I expect this to go smoothly

**Pull request merged**